Java/Java.sublime-syntax

```diff
@@ -253,25 +373,33 @@ contexts:
       scope: keyword.operator.arithmetic.java
     - match: (!|&&|\|\|)
       scope: keyword.operator.logical.java
-    - match: (?<=\S)\.(?=\S)
-      scope: keyword.operator.dereference.java
+    - match: \.
+      scope: punctuation.accessor.java
```

Comment:
Great. Some suggestions from my side.

`.` → dot
`::` → double-colon
`->` → arrow
`=>` → fat-arrow ? (probably not used as an accessor anywhere but whatever)
`\` → backslash (yes, Moonscript uses this...)